Packaging/Package.appxmanifest

```diff
@@ -81,23 +95,11 @@
           </com:ComServer>
         </com:Extension>
         <uap3:Extension Category="windows.appExtension">
-          <uap3:AppExtension Name="com.microsoft.devhome" Id="PG-SP-ID1" PublicFolder="Public" DisplayName="ms-resource:AppDisplayNameCoreExt" Description="ms-resource:AppDescriptionCoreExt">
+          <uap3:AppExtension Name="com.microsoft.devhome" Id="PG-SP-ID1" PublicFolder="Public" DisplayName="ms-resource:AppDisplayNameDev" Description="ms-resource:AppDescription">

```

Comment:
I think there may have been some merge conflicts in this file from here down, widget stuff should go back.

Reply:
ah thanks for catching this, the build.ps1 changes some of this every time its run and doesn't put it back.